lib/all_futures/base.rb

```diff
@@ -3,9 +3,13 @@
 module AllFutures
   class Base < ActiveEntity::Base
     prepend ::AllFutures::Callbacks
-    include ::ActiveModel::Conversion
     include ::AllFutures::Persist
     include ::AllFutures::Dirty
+    include ::AllFutures::Timestamp
+    include ::ActiveModel::Conversion
+    extend ::ActiveModel::Naming
```

Comment:
What happens when we extend `ActiveModel::Naming`?

Reply:
That's the most innocent module inclusion, I think.

It adds methods like `self.model_name`, `self.plural` etc.... stuff we'd probably need anyway even if we were to implement our own version of `cache_key`